screen/model/media/mediahandler.js

```diff
@@ -0,0 +1,37 @@
+
+module.exports = class MediaHandler {
+
+    constructor(id, dom){
+        this.id = id;
+        this.ui = this._createMediaWrapper(dom);
+    }
+
+    handleMessage(msg){
+        switch(msg.command){
+            case "create":
```

Comment:
I think the inheritance would be easier to follow if the things all handlers need (create, destroy) were methods the command router could call instead of messages handled inside handleMessage.